src/components/Navbar&Footer/Layout.css

```diff
@@ -42,7 +42,7 @@
 }
 
 .dropdown-content a:hover {
-    color: lightgreen;
+    color: #c2cffc;
```

Comment:
we can combine this with the nav-link hover section at the top, try: `.nav-link:hover, .dropdown-content a:hover{...}`